src/pretix/base/settings.py

```diff
@@ -1190,6 +1190,14 @@ def country_choice_kwargs():
                         "Defaults to your event name."),
         )
     },
+    'mail_sales_channel_placed_paid': {
+        'default': ['web'],
+        'type': list,
+    },
+    'mail_sales_channel_ticket_reminder': {
+        'default': ['web'],
+        'type': list,
```

Comment:
`List[str]`?

Reply:
``list`` is correct, ``List[str]`` only works as a type annotation, while this needs to be an actual class that hierarkey can unserialize to.